src/Service/ElasticsearchService.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file
+ * Contains service to integrate with elasticsearch.
```

Comment:
Found out the in Drupal 8 the file comment content is moved into the `@class` description above the class. Not sure what the standard is for symfony... and have not found a clear answer on google.

Reply:
Will leave it as is atm.